lib/setupmanagers/qemuhck/devices/vhost-user-fs-pci.json

```diff
@@ -12,7 +12,8 @@
   "pre_start_commands": [
     "mkdir -p @fs_daemon_share_path@",
     "rm -f @fs_daemon_socket@",
-    "@fs_daemon_bin@ --socket-path=@fs_daemon_socket@ -o source=@fs_daemon_share_path@ -o cache=always &"
+    "@fs_daemon_bin@ --socket-path=@fs_daemon_socket@ -o source=@fs_daemon_share_path@ --cache always &",
+    "bash -c 'until [ -S \"@fs_daemon_socket@\" ]; do echo \"Socket does not exist, waiting...\"; sleep 1; done'"
```

Comment:
I didn't know pre_start_commands are not executed in bash.
It's not good to use a different shell only here so let's revert to `while`.

Reply:
I used the same structure for `while`. We can just drop `bash -c `

Reply:
Oh, I see. I have just confirmed `until` is available for the POSIX shell too.